bim2sim/sim_settings.py

```diff
@@ -1,5 +1,6 @@
 """Module for defining simulation model specific process settings.
-This targets both, settings to set for the later simulation and settings for the
+This targets both, settings to set for the later simulation and settings for
```

Comment:
Please fix the linebreaks in this file

Reply:
fixed with afaf6f694fbadc8c52ca8cd5c5bf7dbc258ce124